### PR TITLE
Output the number of read threads by a user in social stats.

### PR DESCRIPTION
**Pull request by Kelketek**

### CODE REVIEW

**Comment by e-kolpakov** [comment]
@Kelketek :+1: Can't verify locally though - I've switched to birch-rebased edx-solutions fork and looks like there're some one-way mongo migration, so I can't switch to current edx-solutions/master based branches without rebuilding devstack. Please make another verification run for me :)

**Comment by e-kolpakov** [comment]
@Kelketek if you managed to notice my previous message here - disregard that, posted to wrong window.

Looks good :+1:

File: api/users.rb [hunk 1]
```diff
@@ -90,26 +90,33 @@
...
-        "num_comments_generated" => 0
+          "num_threads" => 0,
+          "num_comments" => 0,
+          "num_replies" => 0,
+          "num_upvotes" => 0,
+          "num_downvotes" => 0,
+          "num_flagged" => 0,
+          "num_comments_generated" => 0,
+          "num_thread_followers" => 0,
+          "num_threads_read" => 0,
```
Comment (smarnach) [inline api/users.rb]: Minor nit: The new indentation seems to be inconsistent with the rest of the file (everything else seems to use two spaces).

**Comment by smarnach** [comment]
The code changes look pretty straight-forward and good :+1:
I didn't find time to test the changes myself, though.  There are currently too many things I need to figure out.

**Comment by Kelketek** [comment]
@e-kolpakov The issue was pre-existing. But I have fixed it now. Merging.

**Pull request merged**